packages/components/src/Toggle/Toggle.scss

```diff
@@ -0,0 +1,75 @@
+// all
+$toggle-height: 20px;
+
+$toggle-track-height: 10px;
+$toggle-track-width: 40px;
+$toggle-thumb-size: 20px;
+
+.tc-toggle {
+	display: flex;
+	height: $toggle-height;
+	line-height: $toggle-track-height;
+	padding: ($toggle-height - $toggle-track-height) / 2 0;
+
+	.tc-toggle-model {
+		display: none;
+
+		&:checked ~ .tc-toggle-element {
+			background: rgba($scooter, 0.6);
+
+			&:after {
+				left: $toggle-track-width - $toggle-thumb-size;
+				background: $scooter;
+			}
+		}
+
+		&:disabled ~ .tc-toggle-element {
+			background: rgba($dove-gray, 0.25);
+			cursor: not-allowed;
+
+			&:after {
+				background: $concrete;
+			}
+		}
+
+		&:disabled ~ .tc-toggle-status-label {
+			color: rgba(0, 0, 0, 0.6);
+		}
+	}
+
+	.tc-toggle-element {
+		position: relative;
+		display: block;
+		height: $toggle-track-height;
+		width: $toggle-track-width;
+		background: rgba($dove-gray, 0.6);
+		border-radius: 100px;
```

Comment:
You should use a variable like $toogle-border-radius since it is used several times. I think 20px is fine.